src/components/Charts/SparklineChart.tsx

```diff
@@ -37,6 +38,7 @@ function SparklineChart({ width, height, tokenData, pricePercentChange, timePeri
       getX={(p: PricePoint) => widthScale(p.timestamp)}
       getY={(p: PricePoint) => rdScale(p.value)}
       curve={curveCardinal.tension(curveTension)}
+      marginTop={marginTop}
```

Comment:
can you just add `5` here instead of adding the prop?

Reply:
sgtm done